dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -153,14 +153,11 @@ def _run_test(
     shell_params: ShellParams,
     extra_pytest_args: tuple,
     python_version: str,
-    db_reset: bool,
     output: Output | None,
     test_timeout: int,
     output_outside_the_group: bool = False,
     skip_docker_compose_down: bool = False,
 ) -> tuple[int, str]:
-    shell_params.run_tests = True
```

Comment:
Turned out that those shell_params should be set in "shell_params" outside - otherwise they were missing in `docker pull` and cache envs did not contain RUN_TESTS and DB_RESET set to True.